docs/variants/pc_engines/initial-deployment.md

```diff
@@ -0,0 +1,85 @@
+# Initial deployment
+
+This document describes the process of updating
+[PC Engines coreboot firmware](https://pcengines.github.io/) to Dasharo firmware
+based on EDK II.
+
+## OS compatibility
+
+Switching from BIOS to UEFI-based firmware might cause compatibility issues if
+you already had an operating system installed on your platform. Some OSs can
+handle the switch without any issues, while others may need to be reinstalled.
+We have tested the update on several operating systems. The results are
+presented in the table below:
+
+| OS | Can be booted from UEFI  |
+|----|------------------------- |
```

Comment:
We should just link to the test results' spreadsheet, so we don't repeat test results here.

Reply:
Can we link to the specific cell in the spreadsheet? We needed a simple information to the users, on which OSes can you safely transition and still use your OS, on which not.